### PR TITLE
Added Yuan QCap

### PR DESCRIPTION
**Pull request by jjomier**

### CODE REVIEW

File: applications/endoscopy_tool_tracking/python/endoscopy_tool_tracking.py [hunk 1]
```diff
@@ -44,9 +45,9 @@ def __init__(self, data, record_type=None, source="replayer"):
         record_type : {None, "input", "visualizer"}, optional
             Set to "input" if you want to record the input video stream, or
             "visualizer" if you want to record the visualizer output.
-        source : {"replayer", "aja"}
+        source : {"replayer", "aja", "yuan"}
```
Comment (wendell-hom) [inline applications/endoscopy_tool_tracking/python/endoscopy_tool_tracking.py:48]:
```suggestion
        source : {"replayer", "aja", "deltacast", "yuan"}
```

File: applications/endoscopy_tool_tracking/python/README.md [hunk 2]
```diff
@@ -50,3 +51,9 @@ the working directory.
     cd <HOLOHUB_BUILD_DIR>
     python3  <HOLOHUB_SOURCE_DIR>/applications/endoscopy_tool_tracking/python/endoscopy_tool_tracking.py --source=aja
     ```
+
```
Comment (wendell-hom) [inline applications/endoscopy_tool_tracking/python/README.md:54]: @jjomier is this branch using the latest code?  The instructions for running with deltacast capture card seems to be missing?

Comment (jjomier) [reply applications/endoscopy_tool_tracking/python/README.md:54]: Deltacast is not supported for Python (only C++)

Comment (wendell-hom) [reply applications/endoscopy_tool_tracking/python/README.md:54]: I see, thanks!

**Pull request merged**